scene/3d/audio_stream_player_3d.cpp

```diff
@@ -354,10 +354,21 @@ StringName AudioStreamPlayer3D::_get_actual_bus() {
 	return internal->bus;
 }
 
+static void apply_max_volume_from_vector(Vector<AudioFrame> &tgt_volume_vector, const Vector<AudioFrame> &src_volume_vector) {
+	for (int64_t i = 0; i < tgt_volume_vector.size(); i++) {
+		const auto frame = AudioFrame(
```

Comment:
```suggestion
		const AudioFrame frame = AudioFrame(
```
`auto` is [not allowed](https://contributing.godotengine.org/en/latest/engine/guidelines/cpp_usage_guidelines.html#auto-keyword)